BigQuery/tests/System/ManageJobsTest.php

```diff
@@ -109,4 +109,22 @@ public function testReloadsJob($job)
     {
         $this->assertEquals('bigquery#job', $job->reload()['kind']);
     }
+
+    public function testListsJobsWithParent()
+    {
+        $query = self::$client->query('SELECT "a";SELECT "b";');
+        $job = self::$client->startQuery($query);
+        $jobId = $job->id();
+
+        // wait a second for the children to become available.
+        sleep(1);
```

Comment:
the sleep seems odd, there's not a wait until done method for the job?

Reply:
Oh, there definitely is, and I sure feel dumb that I didn't use it here. Thanks for the catch. :)